themes/mongodb/src/navbar/navbar.js

```diff
@@ -187,7 +187,7 @@ class Navbar extends preact.Component {
             <nav className="navbar">
                 <div className="navbar__left">
                     <a href="https://www.mongodb.com/">
-                        <img src={`${baseUrl}/images/mongodb-logo.png`} className="navbar-brand" alt="MongoDB Logo" />
+                        <img src={'snooty-koueq.mongodbstitch.com/logo-mongodb.png'} className="navbar-brand" alt="MongoDB Logo" />
```

Comment:
this should include the protocol, e.g. https://snooty-koueq.mongodbstitch.com/logo-mongodb.png

Additionally, because this is no longer referencing a variable, I think we can remove the curly braces around it?